config/crd/bases/infrastructure.cluster.x-k8s.io_linodevpcs.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.14.0
+    controller-gen.kubebuilder.io/version: v0.13.0
```

Comment:
might need to update devbox dependencies

Reply:
Isn't this a Go dependency?

Reply:
if you pull in the latest from `main` and regenerate there is a fix in the Makefile. basically devbox doesn't have the latest controller-tools version

Reply:
Do you mean regenerate with `make manifests` (and controller-gen `v0.14.0`) or just run the controller-gen (`v1.13.0`) binary provided by Devbox?

Reply:
if you rerun `make manifests` or `make build` from main it *should* upgrade things back to 0.14.0

Reply:
I should be already using controller-gen `v0.14.0` for these specific changes 🤔

Reply:
in your latest commit I still see the `v0.13.0` label on the resources

Reply:
No, I think that's the suggested change?